scalafix-rules/src/main/scala/scalafix/internal/rule/RemoveUnused.scala

```diff
@@ -146,4 +172,17 @@ class RemoveUnused(config: RemoveUnusedConfig)
     case i: Defn.Def => Some(i.tokens)
     case _ => None
   }
+
+  private def posExclParens(tree: Tree): Position = {
+    val leftTokenCount =
+      tree.tokens.takeWhile(tk => tk.is[Token.LeftParen] || tk.is[Trivia]).size
+    val rightTokenCount = tree.tokens
+      .takeRightWhile(tk => tk.is[Token.RightParen] || tk.is[Trivia])
+      .size
+    tree.pos match {
+      case Position.Range(input, start, end) =>
+        Position.Range(input, start + leftTokenCount, end - rightTokenCount)
+      case other => other
+    }
+  }
```

Comment:
This is a workaround to match a Position with unused diagnostics, if a patvar is embraces by `()`,  like `(s: String)`.
I'd appreciate it if  your can advice me a smarter way.

Reply:
Looks good to me!

Reply:
Thank you for your confirmation!